kubernetes/overlays/agreement-readmodel-writer/dev-refactor/configmap.yaml

```diff
@@ -0,0 +1,9 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{SERVICE_NAME}}
+  namespace: {{NAMESPACE}}
+data:
+  KAFKA_CLIENT_ID: "{{NAMESPACE}}-pagopa-interop-agreement"
+  KAFKA_GROUP_ID: "{{NAMESPACE}}-agreement-readmodel-writer"
+  AGREEMENT_TOPIC: "{{ARGEEMENT_TOPIC}}"
```

Comment:
```suggestion
  AGREEMENT_TOPIC: "{{AGREEMENT_TOPIC}}"
```

Reply:
I had to stare the suggestion for 30 seconds to spot the typo 😂 thanks!